README.md

```diff
@@ -205,7 +205,7 @@ The `xrpl-py` library automatically populates the `fee`, `sequence` and `last_le
 
 ```py
 from xrpl.models.transactions import Payment
-from xrpl.transaction import send_reliable_submission, safe_sign_and_autofill_transaction
+from xrpl.transaction import send_reliable_submission, sign_and_autofill
```

Comment:
IMO we should recommend that people sign and autofill separately - then, they're not passing in a client in the same method that they're signing.